app/keepers/keepers.go

```diff
@@ -682,6 +694,16 @@ func NewAppKeeper(
 		appKeepers.IBCFeeKeeper,
 	)
 
+	// Prepare the osmosis ibc hooks logic
+	milkPrefix := sdk.GetConfig().GetBech32AccountAddrPrefix()
+	wasmHooks := ibchooks.NewWasmHooks(appKeepers.IBCHooksKeeper, &appKeepers.WasmKeeper, milkPrefix) // The contract keeper needs to be set later
```

Comment:
What does this comment mean?

Reply:
Forgot to remove after taking the code from osmosis :see_no_evil: